client/nginx.conf

```diff
@@ -0,0 +1,52 @@
+# nginx.conf
+worker_processes auto;
+
+pid /tmp/nginx.pid;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  include /etc/nginx/mime.types;
+  client_body_temp_path /tmp/client_temp;
+  proxy_temp_path       /tmp/proxy_temp_path;
+  fastcgi_temp_path     /tmp/fastcgi_temp;
+  uwsgi_temp_path       /tmp/uwsgi_temp;
+  scgi_temp_path        /tmp/scgi_temp;
+
+  server {
+  listen 8080;
+  server_name _;
+
+  index index.html;
+  error_log /dev/stdout info;
+  access_log /dev/stdout;
+  root /app;
+
+  location /namerequest {
+    alias /app;
+    try_files $uri /index.html;
+    gzip on;
+    gzip_vary on;
+    gzip_min_length 10240;
+    gzip_proxied any;
+    gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
+    gzip_disable "MSIE [1-6]\.";
+  }
+
+  # For status of ngnix service, OpenShift is configured to call this
+  location /nginx_status {
```

Comment:
This section is missing from some other UI projects (eg, bcrs-business-create-ui). Is that OK?